src/components/Summary.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import _ from 'lodash';
+
+import SummaryScope from './SummaryScope.js';
+
+const Summary = React.createClass({
+  getInitialState() {
+    return {summary: []};
+  },
+  componentDidMount() {
+    this.fetchScore.call(this);
```

Comment:
Sounds like a good practice (https://facebook.github.io/react/docs/component-specs.html#mounting-componentwillmount) if only to optimize for a few cycles.